test/passing/open-auto.ml.ref

```diff
@@ -232,18 +234,20 @@ let _ =
   let open struct
     type t
   end in
-  let open functor
-             (A : S)
-             ->
-             struct
-               type t
-             end in
-  let open functor
-             (_ : S)
-             ->
-             struct
-               type t
-             end in
+  let open
+        functor
```

Comment:
Is this expected ? (not related to the issue linked in this PR)
I'd expect the `let open` to not break but the functor indented only by two spaces.

Reply:
Yes for now "let open" is considered being in single keyword. It didn't shock anyone so far regarding the lack of break but we can definitely change that in a future PR.

Reply:
Good catch, I didn't notice the indentation, I will have a look.